javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,63 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+function renderProducts(availableProducts) {
+    availableProducts.forEach(product => {
+        /*creating elements*/
+        let productLi = document.createElement('li');
+        let productName = document.createElement('h2');
+        let productPrice = document.createElement('p');
+        let productRating = document.createElement('p');
+        /*setting innerHTML*/
+        productName.innerHTML = product.name;
+        productPrice.innerHTML = `Price: ${product.price}`;
+        productRating.innerHTML = `Rating: ${product.rating}`;
+        /*appending*/
+        productLi.appendChild(productName);
+        productLi.appendChild(productPrice);
+        productLi.appendChild(productRating)
+        productsUl.appendChild(productLi);
+    });
+}
+
+/*searching by name */
+const searchBox = document.getElementById("productName");
+const priceBox = document.getElementById("max-price");
+console.log(searchBox);
+searchBox.addEventListener("keyup", getProductsByName)
+function getProductsByName() {
+    searchValue = searchBox.value.toLowerCase();
+    const filterProducts = products.filter(product => product.name.toLowerCase().includes(searchValue));
+    productsUl.innerHTML = '';
+    renderProducts(filterProducts);
+}
+/*searching by price */
```

Comment:
Single line comments can be done with
```suggestion
// searching by price
```